tabpfn_client/service_wrapper.py

```diff
@@ -49,21 +49,22 @@ def set_token_by_registration(
         )
         if access_token is not None:
             self.set_token(access_token)
-        return is_created, message
+        return is_created, message, access_token
 
-    def set_token_by_login(self, email: str, password: str) -> tuple[bool, str]:
-        access_token, message = self.service_client.login(email, password)
+    def set_token_by_login(self, email: str, password: str) -> tuple[bool, str, int]:
+        access_token, message, status_code = self.service_client.login(email, password)
 
         if access_token is None:
-            return False, message
+            return False, message, status_code
+        elif status_code == 200:
```

Comment:
Because if it is 200 we need not prompt the user for the verification token as the user is already verified. However, in the case of 403 we need to reverify email in which we send the user a new verification token, which he can enter in the input prompt and then we set the token there.

Reply:
yeah i see that you want to do it that way, but I don't understand why

Reply:
you can merge this even with this open, but i would like to understand whether this is something that we should fix later on or whether there is an actual reason to handle the access_token differently here